server/__db/initial-setup.sql

```diff
@@ -0,0 +1,2 @@
+drop database if exists cyf_test;
```

Comment:
It'd be really helpful to indicate in the file name that this is just a local task and not something that runs on heroku. Maybe `set-up-local-db.sql`.